sign.sh

```diff
@@ -1,4 +1,2 @@
 rm -f signed.apk
 java -jar mstar_system_sign_tool/signapk.jar mstar_system_sign_tool/platform.x509.pem mstar_system_sign_tool/platform.pk8  app/build/outputs/apk/app-debug.apk signed.apk
-
-adb install -r signed.apk
```

Comment:
Also removed this because it didn't seem useful for the install, since everything has to be tested on the device anyway and I just scp'd it across.